popmon/visualization/templates/footer.html

```diff
@@ -9,10 +9,18 @@
 <!-- Bootstrap core JavaScript -->
 <!-- Plugin JavaScript -->
 <!-- Custom JavaScript for this theme -->
+{% if online_report %}
+    <script src="https://cdn.plot.ly/plotly-2.12.1.min.js"></script>
+{% else %}
+    <script>
+        {% include 'assets/js/plotly.js' %}
+    </script>
```

Comment:
All files below are concatenated in the same script tag. For consistency, we should do this for plotly too.